checklist.rst

```diff
@@ -2,12 +2,8 @@
 
 Procedure:
 
-1. check GitHub Actions test results: https://github.com/sphinx-doc/sphinx-intl/actions
-2. update release version/date in ``CHANGES.rst``
-3. ``python -m build``, see details: setup.cfg
-4. ``twine upload dist/<target-package-file>``
-5. check PyPI page: https://pypi.org/p/sphinx-intl
-6. tagging with version name that MUST following semver. e.g.: ``git tag 1.0.1``
-7. ``git push --tags`` to push tag
-8. bump version in ``sphinx_intl/__init__.py`` and ``CHANGES.rst`` then commit/push
-   them onto GitHub
+1. update release version/date in ``CHANGES.rst``
+2. create GitHub Release with new version tag, it will create a release on PyPI.
+   tag MUST following semver. e.g.: ``2.3.1``
+3. check PyPI page: https://pypi.org/p/sphinx-intl
+4. bump version in ``CHANGES.rst`` then commit/push it onto GitHub
```

Comment:
Mention that pushing will trigger the release action?

Reply:
No, it is assumed that the release action is not triggered by push. 
The release action occurs in step 2.
Step 4 refers to preparing a next release section in the CHANGES.rst.